dev/ci/user-overlays/08985-ejgallego-build+pack_gramlib.sh

```diff
@@ -0,0 +1,6 @@
+if [ "$CI_PULL_REQUEST" = "8985" ] || [ "$CI_BRANCH" = "build+pack_gramlib" ]; then
+
+    Elpi_CI_REF=use_coq_gramlib
+    Elpi_CI_GITURL=https://github.com/ejgallego/coq-elpi
```

Comment:
I guess I prefer to link elpi against camlp5 (as a package) while I find the time to use another parsing engine.
Also I'm not so sure what happens here: `CoqMakefile.in` does not prevent me from linking `camlp5.gramlib` anymore. So I guess it is linked. But the PR adds `open Gramlib` as if you wanted elpi to use the Coq copy of gramlib. It the overlay really necessary?

Reply:
Unless I am mistaken you are still linking elpi to camlp5, aren't you? But elpi seems to be also using the parsing engine of Coq, then you need to use Coq's gramlib.

Reply:
Oh, I see, I'll look again at your PR